ui/src/message/MessagesStore.ts

```diff
@@ -159,5 +160,6 @@ export class MessagesStore {
         hasMore: true,
         nextSince: 0,
         loaded: false,
+        isLoading: false,
```

Comment:
Why not use the existing `loaded` boolean?

Reply:
Good question. Looks like I completely missed that. I thought this was only used for initial loading. Reverted addition of `isLoading` in aee7ef0f50e69606cb538e7ce866d7aa9b7141ad.